packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx

```diff
@@ -40,6 +57,29 @@ describe('CreateImageTab', () => {
     expect(submitButton).toBeEnabled();
   });
 
+  it('should pre-fill Linode and Disk from search params', async () => {
+    const linode = linodeFactory.build();
+    const disk = linodeDiskFactory.build();
+
+    mockLocation.search = `selectedLinode=${linode.id}&selectedDisk=${disk.id}`;
+
+    server.use(
+      http.get('*/v4/linode/instances', () => {
+        return HttpResponse.json(makeResourcePage([linode]));
+      }),
+      http.get('*/v4/linode/instances/:id/disks', () => {
+        return HttpResponse.json(makeResourcePage([disk]));
+      })
+    );
+
+    const { getByLabelText } = renderWithTheme(<CreateImageTab />);
```

Comment:
Can we remove all of the `react-router-dom` mocking? We have an existing helper we can use

```suggestion
const { getByLabelText } = renderWithTheme(<CreateImageTab />, {
      MemoryRouter: {
        initialEntries: [
          `/images/create?selectedLinode=${linode.id}&selectedDisk=${disk.id}`,
        ],
      },
    });```